_includes/usagepolicy.html

```diff
@@ -4,23 +4,23 @@
       <p>This page contains the policy for the Red Hat sponsored anonymous data collection. See <a href="/usage">Usage</a> page for details.</a></p>
       <p class="intropara">In connection with your use of this Red Hat offering, Red Hat may collect telemetry data about your use of the software. This data allows Red Hat and community extension writers to understand how the software is being used and to improve Red Hat offerings and support, including identifying, troubleshooting, and responding to issues that impact users. The data may be shared internally within Red Hat to improve the user experience.</p>
       <H3>What information does Red Hat collect?</H3>
-      <p>Tools within the software monitor various metrics and this information is transmitted to Red Hat.  Metrics include the following information such as:</p>
+      <p>Tools within the software monitor various metrics and this information is transmitted to Red Hat. Metrics include the following information:</p>
       <ul>
         <li>Usage information about components, features and extensions.</li>
         <li>The number and nature of errors that occur during development.</li>
         <li>Quarkus version</li>
         <li>Java version, vendor</li>
         <li>Build tool used</li>
         <li>Build duration</li>
-        <li>List of extensions and their versions</li>
-        <li>Timezone</li>
+        <li>List of <b>community</b> extensions and their versions</li>
```

Comment:
since we also report productized extensions would something like this be better:

```suggestion
        <li>List of <b>public</b> extensions and their versions</li>
```